### PR TITLE
Created template for the jobs 32-bit-build-smoke, fedora-smoke and fr…

### PR DESCRIPTION
**Pull request by rakshithakamath94**
Hi @mscherer, 

Please do review this PR.

Thank you.

### CODE REVIEW

File: build-gluster-org/jobs/smoke-template.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          timeout: 45
+          script_name: freebsd-smoke 
+
+    jobs:
+      - 'gh_{os_type}-{name}'
+
+- job-template:
+    name: 'gh_{os_type}-{name}'
+    node: '{node_name}'
+    description: '{description}'
```
Comment (mscherer) [inline build-gluster-org/jobs/smoke-template.yml:31]: Could we automatically add the link to the script here ? Since they are all the same, except script name, this would reduce cut and paste.

File: build-gluster-org/jobs/smoke-template.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
+- project:
+    name: 'smoke'
+    os_type:
+      - 32-bit-build:
+          node_name: fedora32
+          description: smoke job for 32 bit build - https://github.com/gluster/build-jobs/blob/main/build-gluster-org/scripts/32-bit-build-smoke.sh
+          keyword: 32-bit-build 
+          timeout: 30
```
Comment (mscherer) [inline build-gluster-org/jobs/smoke-template.yml:8]: Is there a way to get a default value for the timeout ? So we will add it only when it is different from the default, like blacklist_branch

File: build-gluster-org/jobs/clang-format.yml [hunk 3]
```diff
@@ -4,6 +4,7 @@
     description: Pre-commit clang-format check - https://github.com/gluster/build-jobs/blob/main/build-gluster-org/scripts/clang-format.py
     project-type: freestyle
     concurrent: true
+    disabled: true
```
Comment (mscherer) [inline build-gluster-org/jobs/clang-format.yml:7]: This should be a in a different PR

**Comment by mscherer** [comment]
Seems good, but could we have differents PR for the different tickets opened, so we can review them and merge them independenly ?

**Comment by rakshithakamath94** [comment]
Sure

File: build-gluster-org/jobs/smoke-template.yml (hunk 2 above)
Comment (rakshithakamath94) [reply build-gluster-org/jobs/smoke-template.yml:8]: Yes, I will update it

File: build-gluster-org/jobs/clang-format.yml (hunk 3 above)
Comment (rakshithakamath94) [reply build-gluster-org/jobs/clang-format.yml:7]: Ok, I will have it removed from this PR

File: build-gluster-org/jobs/smoke-template.yml (hunk 1 above)
Comment (rakshithakamath94) [reply build-gluster-org/jobs/smoke-template.yml]: Updated the description.

**Review by deepshikhaaa — APPROVED** [review]
